addons/vz99/functions/fnc_canDeployWeapon.sqf

```diff
@@ -0,0 +1,18 @@
+/*
+ * Author: PabstMirror
+ *
+ *
+ * Arguments:
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Example:
+ * [] call 
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+((secondaryWeapon ACE_player) == QGVAR(carryWeapon)) 
```

Comment:
Do we want to limit this a bit more? Like if there's no room, or if they're on a 60 degree incline?
Also the header is a bit jacked up, Return value and example are incorrect.